### ISSUE TITLE
Doesn't seem to be working with the current version of Firefox

### ISSUE THREAD

**Issue by Freak0104** [rimutaka/spotify-play-all-saved-albums#1]
The button appears, it's clickable, but it never starts processing.

**Comment by rimutaka** [comment]
@Freak0104 , yes, I am aware about it. They probably changed the page layout. I'll take a look, but that won't be this week.

**Comment by hotvomit** [comment]
Seems to not be working again??

**Comment by rimutaka** [comment]
Yes, the UI changed and I had no time to chase it up.

**Comment by hotvomit** [comment]
dang that sucks

**Comment by hotvomit** [comment]
> Yes, the UI changed and I had no time to chase it up.

Still busy?
